data/sql/updates/pending_db_world/rev_1586845118268975400.sql

```diff
@@ -0,0 +1,90 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1586845118268975400');
+
+ALTER TABLE `holiday_dates` ADD COLUMN `holiday_duration` INT(10) UNSIGNED DEFAULT 0 NOT NULL AFTER `date_value`;
+UPDATE `holiday_dates` SET `holiday_duration`= 360 WHERE `id`= 327;
+
+-- Darkmoon Faire
+UPDATE `game_event` SET `start_time`= '2020-01-05 00:01:00', `end_time`= '2030-01-01 06:00:00' WHERE `eventEntry`= 3;
```

Comment:
and here delete ``, `end_time`= '2030-01-01 06:00:00'``

Reply:
Ah totally missed it, will fix later